app/services/authorizer.rb

```diff
@@ -19,6 +19,7 @@ def can?(permission, subject = nil)
 
   def find_collection(resource_class, options = {})
     permission = options.delete :permission
+    Foreman::Logging.logger('permissions').debug "checking permission #{permission}"
 
     # retrieve all filters relevant to this permission for the user
     base = user.filters.joins(:permissions).where(["#{Permission.table_name}.resource_type = ?", resource_name(resource_class)])
```

Comment:
that might be a big list which does not say much but I'll try it, if we get some useful information without loading additional resources it might be interesting (at least role_id)